coursefunctions.h

```diff
@@ -167,49 +200,49 @@ void TestDrawTriangle(Buffer2D<PIXEL> & target)
         *************************************************/
         Vertex verts[3];
         Attributes attr[3];
-        verts[0] = {100, 362, 1, 1};
-        verts[1] = {150, 452, 1, 1};
-        verts[2] = {50, 452, 1, 1};
+        verts[0] = (Vertex){100, 362, 1, 1};
+        verts[1] = (Vertex){150, 452, 1, 1};
```

Comment:
Our submissions differ in that you changed the code inside the TestDrawPixel() method.